docs/configuration.rst

```diff
@@ -6,8 +6,8 @@ This page describes how to configure ``pydistcheck``.
 ``pydistcheck`` combines different sources of configuration in the following order.
 
 1. default values
-2. :ref:`pyproject-toml` (or custom TOML file passed via ``--config``)
-3. :ref:`cli-arguments`
+2. `pyproject-toml`_ (or custom TOML file passed via ``--config``)
+3. `cli-arguments`_
```

Comment:
Reference on this: https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#implicit-targets